blog/_posts/2022-03-30-tilt-news-march-2022.md

```diff
@@ -0,0 +1,56 @@
+So, we’re excited to congratulate [Okteto on their Series A](https://www.okteto.com/blog/a-15m-series-a-to-disrupt-modern-development/) and [Signadot](https://www.prnewswire.com/news-releases/y-combinator-alum-signadot-raises-4m-led-by-redpoint-ventures-to-reimagine-microservices-testing-on-kubernetes-301488164.html) on their Seed funding!
+
+## Optimize your Tilt experience
+
+### ...running containers locally
+Many of our users have been experimenting with all kinds of local container orchestration tools to suit their individual needs. Since we last wrote about Rancher Desktop, a lot has changed. In this blog post, Milas gives us a [rundown of different container runtimes with Rancher Desktop](https://blog.tilt.dev/2022/03/04/rancher-desktop-container-runtimes.html).
+
+### ...sharing Tiltfile best practices
+Our vision for Tilt is to make developers’ lives easier. But we don’t stop at individual engineers. Because we believe that sharing is caring, Nick has written this blog post to explain how you can share your custom Tilt magic in a [private extensions library](https://blog.tilt.dev/2022/03/01/dev-env-libraries.html).
+
+### ...running subsets of resources
+Just a couple of weeks ago, we released the [Disable Resources Feature](https://docs.tilt.dev/disable_resources.html). With that in place, teams and individual devs can now [create a catalog of resources](https://blog.tilt.dev/2022/03/03/resource-catalog.html), tailored to their specific requirements and workflows. Spinning up only the resources you need, while still being able to bring up disabled resources with just one click is now a breeze.
+
+### ...managing Pulumi resources
+One of the reasons why we think Tilt is great, is because it easily integrates with other tools. With the new Pulumi extension, it’s no big deal to [manage Pulumi resources from your Tiltfile](https://blog.tilt.dev/2022/03/23/pulumi.html)!
+
+### ...and writing Tiltfiles
+Working with your Tiltfile should be fun, not tiresome. To that end, we recently released the [Snippet Library](https://docs.tilt.dev/snippets.html) to get you started in no time.  
+But also, more seasoned Tilters should not have to remember or constantly look up function signatures and the likes. Therefore, we are proud to present our [official VS Code extension for Tiltfiles](https://marketplace.visualstudio.com/items?itemName=tilt-dev.Tiltfile).  The extension is currently still in alpha stage, so you might encounter some issues here and there. We appreciate all bug reports and feature requests to our [GitHub repo](https://github.com/tilt-dev/vscode-tilt).
+
+## Behind the scenes
+
+Once in a while, we get together virtually to share some laughs and play some games. This time we tried a remote version of the telephone game in which each player has to either paint from a written prompt or describe the image they’re given.  
```

Comment:
aren't you going to link to the game????

Reply:
and maybe recommend it to other remote teams???

Reply:
Oh yeah, good point 😅